chart/templates/mayastor/obs/stats-deployment.yaml

```diff
@@ -0,0 +1,49 @@
+{{- if .Values.stats.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}-stats-aggregator
+  labels:
+    app: stats-aggregator
+    {{ include "label_prefix" . }}/release: {{ .Release.Name }}
+    {{ include "label_prefix" . }}/version: {{ .Chart.Version }}
+spec:
+  replicas: 1
```

Comment:
nit: We might have to add a TODO to handle the case where more than one replicas of this exists and both try to patch the configmap (this is low priority though IMO).

Reply:
noted. can be taken in another PR